app/auth/nsec.tsx

```diff
@@ -181,32 +190,41 @@ export default function Login() {
                   {npub.slice(npub.length - 7, npub.length)}
                 </Text>
               )}
-              {loading ? (
-                <ActivityIndicator animating={loading} size="small" />
-              ) : (
-                <View
-                  style={{
-                    display: "flex",
-                    flexDirection: "row",
-                    alignItems: "center",
-                    gap: 6,
-                  }}
-                >
-                  {profileEvent?.picture && (
-                    <Avatar size={20} imageUrl={profileEvent.picture} />
-                  )}
-                  {profileEvent?.name && (
-                    <Text
-                      style={{
-                        fontSize: 14,
-                        color: colors.text,
-                      }}
-                    >
-                      {profileEvent.name}
-                    </Text>
-                  )}
-                </View>
-              )}
+              <View
+                style={{
+                  display: "flex",
+                  flexDirection: "row",
+                  alignItems: "center",
+                  gap: 6,
+                }}
+              >
+                {loading ? (
+                  <>
+                    <ActivityIndicator animating={true} size={npubAvatarSize} />
+                    <Text>Searching for profile...</Text>
+                  </>
+                ) : (
+                  <>
+                    {profileEvent?.picture && (
+                      <Avatar
+                        size={npubAvatarSize}
+                        imageUrl={profileEvent.picture}
+                      />
+                    )}
+                    {profileEvent?.name && (
+                      <Text
+                        style={{
+                          fontSize: 14,
+                          color: colors.text,
+                        }}
+                      >
+                        {"Some fake name placeholder that is super long" ||
```

Comment:
Can probably remove this line
```suggestion
                        {"Some fake name placeholder that is super long" ||
```